src/SensioLabs/Melody/Tests/IntegrationTest.php

```diff
@@ -118,6 +118,18 @@ public function testRunWithPhpOptions()
         $this->assertContains('memory_limit=42M', $output);
     }
 
+    public function testRunWithForkRepositories()
```

Comment:
I would like a better test and use a fork repository with real difference. I tried to zip a Melody git repository with only the class Melody and the Melody::VERSION changed. The test unziped the repository in a vfsstream directory but of cousre Composer didn't recognize vfs protocol